pkg/logger/logger.go

```diff
@@ -222,8 +250,21 @@ func (l *Logger) PrintReq(username, upstream string, req *http.Request, url url.
 		UserAgent:       fmt.Sprintf("%q", req.UserAgent()),
 		Username:        username,
 	})
+	// Fallback for template errors
+	if err != nil {
+		_, ferr := fmt.Fprintf(l.writer, "%s - %s [%s] %s %s %s %q %s %q %d %d %0.3f",
+			client, username, FormatTimestamp(ts), req.Host, req.Method, upstream,
+			url.RequestURI(), req.Proto, req.UserAgent(), status, size, duration)
+		if ferr != nil {
+			panic(ferr)
+		}
+		panic(err)
```

Comment:
Why do we panic here but not on the others?

Reply:
Because I missed deleting this one 😅 

Good catch, I'll clean it up.

Reply:
Is this all resolved now?

Reply:
Yeah, I confirmed in manual tests that bad templates would fail validation before we ever got to this point.